Moved CleanBuilderTest to package peass/ci/clean since this is wher C…

mawHBT opened the pull request:
CleanBuilderTest was in de.peass.ci.clean.callables, CleanBuilder in de.dagere.peass.ci.clean so I moved CleanBuilderTest to peass.ci.clean.
Packages still differ bei dagere

DaGeRe commented:
Could you than move to the exact package, i.e. including dagere? Or is there a reason to not do so?

mawHBT commented:
All packages in test are missing "dagere", except de.dagere.peass.ci.peassOverview.

DaGeRe commented:
Ok, so if you're starting to change the package, change it to the correct package directly. Since I do not own the domain peass.de, we cannot deploy anything under de.peass and it does not really make sense to have anything in this package. If you want to, you can also change all packages.

mawHBT commented:
I did not create any package (also no peass.de), I just moved CleanBuilderTest one level up. All packages in main are named de.dagere.peass.ci..., all in test are named de.peass.ci...
CleanBuilderTest was in de.peass.ci.clean.callable, so I just moved it up to de.peass.ci.clean since CleanBuilder is in clean also, not in callable.

DaGeRe commented:
Yes, but by convention, one should only use package names associated with the domain one owns (in this case: peass.dagere.de). It does not make sense to move packages one time to de.peass.XX, and afterwards de.dagere.peass.XX, since these two changes would just create noise in the git history. 
To shorten the process, I just reorganized the packages myself, including the move of `CleanBuilderTest`: https://github.com/jenkinsci/peass-ci-plugin/commit/de2949572f5911ba18843c092eac2f019e821a9d Does this work for you? Otherwise, feel free to create another PR.

mawHBT commented:
Packages were reorganized with [de29495](https://github.com/jenkinsci/peass-ci-plugin/commit/de2949572f5911ba18843c092eac2f019e821a9d), including movement of `CleanBuilderTest`. So this PR is omitted.
